### ISSUE TITLE
Installation of azure-cli-ml extension fails on CLI version 2.30.0

### ISSUE THREAD

**Issue by nilushancosta** [Azure/azure-cli#20524]
**Describe the bug**
When using Azure CLI version 2.30.0, installation of [Azure Machine Learning CLI extension v1](https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli) (`azure-cli-ml`) fails with the error `No matching extensions for 'azure-cli-ml'. Use --debug for more information.`. 
Installing the same extension works on Azure CLI version 2.29.2

**To Reproduce**
Execute `az extension add -n azure-cli-ml` on Azure CLI version 2.30.0

**Expected behavior**
Expected the Azure Machine Learning CLI extension v1 to be installed

**Environment summary**
 Install Method - Interactive

`az --version` output
```
azure-cli                         2.30.0

core                              2.30.0
telemetry                          1.0.6

Python location '/opt/az/bin/python3'
Extensions directory '/root/.azure/cliextensions'

Python (Linux) 3.6.10 (default, Oct 29 2021, 10:11:34)
[GCC 9.3.0]

Legal docs and information: aka.ms/AzureCliLegal


Your CLI is up-to-date.

Please let us know how we are doing: https://aka.ms/azureclihats
and let us know if you're interested in trying out our newest features: https://aka.ms/CLIUXstudy
```
OS version - Ubuntu 20.04.3 LTS
Shell type - Bash

**Additional context**

**Comment by yonzhan** [comment]
route to service team

**Comment by paulshealy1** [comment]
Thanks for the feedback! We're currently working on adding support for the CLI v2.30.

**Comment by gkbrida** [comment]
how to install  Azure CLI version 2.29.2 ?

**Comment by Dpetters** [comment]
@gkbrida these are the steps I followed:
1. Download https://raw.githubusercontent.com/Homebrew/homebrew-core/7607de411f8ac0ad926ff2caadf8a9abf713cec8/Formula/azure-cli.rb
2. Run `export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1`
3. Run `brew uninstall azure-cli`
4. Run `brew install ./azure-cli.rb`

**Comment by Dpetters** [comment]
@gkbrida actually I probably only needed the `HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK` step because I had the new `ml` extension already installed.

**Comment by Thrillz** [comment]
> Thanks for the feedback! We're currently working on adding support for the CLI v2.30.

any updates @paulshealy1

**Comment by shirolkar** [comment]
> paulshealy1

@paulshealy1 @Thrillz CLI version 2.31.0 is out and same issue.

**Comment by paulshealy1** [comment]
The current ETA for an updated AzureML CLI is end of next week.

**Comment by csiebler** [comment]
Looks like it works again on the default `ubuntu-20.04` DevOps agent:

```yml
pool:
  vmImage: ubuntu-20.04

steps:
  - task: AzureCLI@2
    displayName: 'Install AML CLI'
    inputs:
      azureSubscription: $(ml_workspace_connection)
      scriptType: bash
      scriptLocation: inlineScript
      inlineScript: |
        az extension add -n azure-cli-ml

```

Shows CLI as:
```
azure-cli                         2.31.0
```

**Comment by paulshealy1** [comment]
The `azure-cli-ml` extension is now available for Azure CLI versions 2.30 and greater.
